templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+{% extends "templates/base.html" %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% block title %}Intel and Canonical{% endblock %}
+
+{% block meta_description %}
+  Intel and Canonical partner to ensure Intel hardware platforms and features are enabled, performant and monitored over time in Ubuntu and other open source software in the Canonical portfolio.
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1GstP9FfZfbXO81YcX-9oCJdP9pamlgpl2kKkSqlrWFE
+{% endblock meta_copydoc %}
+
+{% block outer_content %}
+  {% call(slot) vf_hero(
+    title_text='Choose optimized Linux for Intel
+    ',
+    layout='50/50',
+    is_split_on_medium=true
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        From the desktop and datacenter, to the cloud and the edge, Intel and Canonical collaborate to deliver solutions that enable you to innovate and build at any scale. Our partnership brings together the innovation of open source technology with the peace of mind of comprehensive security.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <!-- This needs to be updated after creation of new form -->
+      <a href="/contact-us" class="p-button--positive">Get in touch</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--3-2 is-highlighted is-cover u-hide--small">
+        {{ image(url="https://assets.ubuntu.com/v1/b42e8e85-hero-img-updated.png",
+                alt="",
+                width="1800",
+                height="1013",
+                hi_def=True,
+                attrs={"class": "p-image-container__image"},
+                loading="auto|lazy") | safe
+        }}
+      </div>
+    {%- endif -%}
+  {% endcall -%}
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>
+            Validated solutions
+            <br class="u-hide--small" />
+            for key verticals
+          </h2>
+        </div>
+        <div class="col">
+          <div class="p-section--shallow">
```

Comment:
This shallow section should be removed since the row is already wrapped in a shallow section. The bottom padding in design is from `p` tag